### DISCUSSION TITLE
Is `main` branch the same as the previous `dev` branch?

### DISCUSSION THREAD

**Discussion by uakihir0** [Netflix/conductor#2424] [Q&A] [answered]
I've seen that there are broken links in the documentation, and I'd like you to fix them, or I'm thinking of doing so, but I'd like to know if I can just replace `dev` with `main` in the branch. Thank you very much.
Is the `main` branch the same as the previous `dev` branch?

For example, `Worker` in the following page.
https://netflix.github.io/conductor/gettingstarted/client/

**Comment by apanicker-nflx** [comment, marked as answer]
@uakihir0 Thanks for pointing out the discrepancies in the documentation. We would really appreciate a PR with the fixes. 

>I can just replace `dev` with `main` in the branch. 
Is the `main` branch the same as the previous `dev` branch?

Yes, the `main` branch can be used as a replacement for the `dev` branch here.
